leaflet/index.d.ts

```diff
@@ -962,6 +962,9 @@ declare namespace L {
         position?: ControlPosition;
     }
 
+    export class Control extends Class {
```

Comment:
You don't need both a class and interface in a case like this. Just move the body of the interface inside the class.